core/src/price_estimation.rs

```diff
@@ -21,7 +21,7 @@ use anyhow::Result;
 use average_price_source::AveragePriceSource;
 use futures::future::{BoxFuture, FutureExt as _};
 use log::warn;
-use price_source::{NoopPriceSource, PriceSource};
```

Comment:
What was `Noop`supposed to mean anyway?

Reply:
https://en.wikipedia.org/wiki/NOP_(code)